railties/lib/rails/application/configuration.rb

```diff
@@ -417,6 +440,14 @@ def content_security_policy(&block)
         end
       end
 
+      # Configures the Permissions-Policy:
+      #
+      #   config.permissions_policy do |policy|
+      #     policy.gyroscope :none
+      #     policy.usb       :self
+      #     policy.autoplay  "https://example.com"
+      #     policy.payment   "https://secure.example.com"
+      #   end
```

Comment:
Similar comment here about https://api.rubyonrails.org/v7.0.2/classes/ActionDispatch/PermissionsPolicy.html doing the heavy lifting.

Reply:
I'm linking to ActionDispatch::PersmissionsPolicy now.